interface/src/app/gene/search/directives.js

```diff
@@ -0,0 +1,321 @@
+angular.module('adage.gene.search.directives', [
```

Comment:
If you replace lines 1-4 into:
```angular.module('adage.gene.search')```
I believe it will get the reference of the module **adage.gene.search** instead of creating a new one. Then you can add directorives or services to it.